lang/ilias_it.lang

```diff
@@ -2727,7 +2726,7 @@ chatroom_adm#:#client_settings#:#Client settings
 chatroom_adm#:#enable_osd#:#Enable On-Screen notifications
 chatroom_adm#:#general_settings_title#:#Impostazioni generali di chat
 chatroom_adm#:#https#:#HTTPS
-chatroom_adm#:#osd_intervall#:#Intervallo di aggiornamento (secondi)
+chatroom_adm#:#osd_interval#:#Intervallo di aggiornamento (secondi)
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.